config/config.txt

```diff
@@ -623,6 +623,12 @@ MOTD motd.txt
 ## but enabled on production (the default).
 CACHE_ASSETS 0
 
+## Loading early assets, primarily used by the preferences menu, greatly slows down init times
+## This dev option here is to bypass that init phase and let the preferences menu queue normally, or overrun the tick to load on demand
+## This should be disabled (through `ENABLE_EARLY_ASSETS 0`) on development,
+## but enabled on production (the default).
+ENABLE_EARLY_ASSETS 0
```

Comment:
Can you explain the thought process of this a bit more?

Also this *needs* to be enabled in the CI config (I think `ciconfig.txt`?). The literal **only** point of early assets is to guarantee that preferences menu assets running before atoms is okay.

Reply:
Explained in below comment, but, to reiterate, the only point of this is to keep the current behaviour of hiding the 12-14s additional init time unless it's needed by a dev (by clicking setup character).

Will add to CI config.